brave-lists/clean-urls.json

```diff
@@ -852,5 +852,16 @@
             "brave_testing1",
             "brave_testing2"
         ]
+    },
```

Comment:
Would you mind moving this up so that we keep our test entries last in the file?

Reply:
Done!